imageset.py

```diff
@@ -19,6 +19,11 @@
 
 ext = boost_adaptbx.boost.python.import_ext("dxtbx_ext")
 
+try:
+    from typing import Iterable, List
+except ImportError:
+    pass
```

Comment:
```suggestion
from typing import Iterable, List
```

I believe typing is always present